manifests/auth_user.pp

```diff
@@ -0,0 +1,23 @@
+# Definition: st2::auth_user
+#
+#  Friendly helper to manage standalone auth for StackStorm
+#
+# Usage
+#
+#  st2::auth_user { 'jfryman':
+#    password => 'neato!',
+#  }
+define st2::auth_user(
+  $ensure   = present,
+  $password = undef,
+) {
+  include ::st2::auth::standalone
+  $_htpasswd_file = $::st2::auth::standalone::htpasswd_file
+
+  httpauth { $name:
```

Comment:
I was just curious what kind of password encryption / hashing mechanism it uses and it looks like it uses and only supports crypt.

That's less than ideal, maybe we should add notice the the docs. Most enterprises will probably use some kind of LDAP or similar auth backend anyway.